plugins/CoreAdminHome/Emails/TokenAuthCreatedEmail.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Emails;
+
+use Piwik\Piwik;
+use Piwik\Plugins\CoreAdminHome\Emails\SecurityNotificationEmail;
+
+class TokenAuthCreatedEmail extends SecurityNotificationEmail
+{
+    protected function getBody()
+    {
+        return Piwik::translate('CoreAdminHome_SecurityNotificationTokenAuthCreatedBody');
+    }
+}
```

Comment:
For this email that says:

```
Someone created an auth token in your account. If it was you, carry on. If you don't recognize this activity, please reset your password.
```

Maybe we should include the name of the token and suggest the user delete it also?